parser_library/src/processing/instruction_sets/asm_processor.cpp

```diff
@@ -765,4 +766,52 @@ void asm_processor::process_CNOP(rebuilt_statement stmt)
     hlasm_ctx.ord_ctx.reserve_storage_area(0, context::alignment { (size_t)*byte_value, (size_t)*boundary_value });
 }
 
+
+void asm_processor::process_START(rebuilt_statement stmt)
+{
+    if (!check(stmt, hlasm_ctx, checker_, *this))
+        return;
+
+    auto sect_name = find_label_symbol(stmt);
+
+    if (std::any_of(hlasm_ctx.ord_ctx.sections().begin(), hlasm_ctx.ord_ctx.sections().end(), [](const auto& s) {
+            return s->kind == context::section_kind::EXECUTABLE || s->kind == context::section_kind::READONLY;
+        }))
+    {
+        add_diagnostic(diagnostic_op::error_E073(stmt.stmt_range_ref()));
+        return;
+    }
+
+    if (hlasm_ctx.ord_ctx.symbol_defined(sect_name))
+    {
+        add_diagnostic(diagnostic_op::error_E031("symbol", stmt.label_ref().field_range));
+        return;
+    }
+
+    auto sym_loc = hlasm_ctx.processing_stack().back().proc_location;
+    sym_loc.pos.column = 0;
+    hlasm_ctx.ord_ctx.set_section(sect_name, context::section_kind::EXECUTABLE, std::move(sym_loc));
+
+    const auto& ops = stmt.operands_ref().value;
+    if (ops.size() != 1)
+        return;
+
+    auto initial_offset = try_get_abs_value(ops.front().get());
+    if (!initial_offset.has_value())
```

Comment:
Emit a diagnostic here that says "operand has to be absolute expression with all symbols previously defined"? It is a special requirement that will probably never be checked in the checker.